dashboard/app/views/devise/registrations/edit.html.haml

```diff
@@ -86,40 +86,44 @@
 
   %div#submit-update= f.submit t('crud.update'), class: 'btn btn-default'
 - if !current_user.teacher_managed_account?
-  %h3= t('user.delete')
+  %h2= t('user.delete')
   %p
     = t('user.delete_text')
     - if current_user.teacher?
       = t('user.delete_text_teacher_note')
     - data_hash = { confirm: t('user.delete_confirm') }
     = button_to t('user.delete'), registration_path(resource_name), data: data_hash, method: :delete, class: 'btn btn-danger'
 - if current_user.can_create_personal_login? && show_take_your_account_with_you
-  %h3= t('user.create_personal_login')
+  %h2= t('user.create_personal_login')
   %p= no_email ? t('user.create_personal_login_under_13_info') : t('user.create_personal_login_info')
   = form_for(current_user, url: '/users/upgrade', html: {id: 'edit_user_create_personal_account'}, namespace: 'create_personal') do |form|
     = form.hidden_field :hashed_email
+    %h3= t('user.enter_new_login_info')
     - if no_email
       .field
         = form.label t('user.create_personal_login_under_13_username')
         = form.text_field :username, maxlength: 255
-      .field
-        = form.label t('user.create_personal_login_under_13_parent_email')
-        = form.text_field :parent_email, maxlength: 255
     - else
       .field
         = form.label :personal_email do
           = t('activerecord.attributes.user.personal_email', url: CDO.studio_url('users/edit/?noEmail=true')).html_safe
         = form.email_field :email, placeholder: '', autocomplete: 'off', maxlength: 255
-    - if current_user.secret_word_account?
-      .field
-        = form.label :secret_words
-        = form.text_field :secret_words, autocomplete: 'off', value: '', maxlength: 255
     .field
       = form.label :password, maxlength: 255
       = form.password_field :password, autocomplete: 'off', maxlength: 255
     .field
       = form.label :password_confirmation
       = form.password_field :password_confirmation, autocomplete: 'off', maxlength: 255
+    - if current_user.secret_word_account?
+      %h3= t('user.confirm_secred_words')
```

Comment:
Grr, I typoed this by accident, I don't know how I didn't catch that. Fixing in another PR